specs/interop/services.md

```diff
@@ -0,0 +1,83 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
```

Comment:
A big thing missing here is the RPC api that i saw was in the PR

Reply:
Added a bit about consuming via a json-rpc server or go library